docs/modelserving/v1beta1/llm/huggingface/multi-node/README.md

```diff
@@ -0,0 +1,315 @@
+
+
+### Example InferenceService
+
+Here’s an example of an `InferenceService` configuration for a Hugging Face model:
+
+```yaml
+apiVersion: serving.kserve.io/v1beta1
+kind: InferenceService
+metadata:
+  name: huggingface-llama3
+spec:
+  predictor:
+    model:
+      modelFormat:
+        name: huggingface
+      storageUri: pvc://llama-3-8b-pvc/hf/8b_instruction_tuned
+    workerSpec: {
+      pipelineParallelSize: 2
+      tensorParallelSize: 1
+    }  
+```
+
+## Serve the Hugging Face LLM Model Using 2 Nodes
+
+Follow these steps to serve the Hugging Face LLM model using a multi-node setup.
+
+### 1. Create a Persistent Volume Claim (PVC)
+
+First, create a PVC for model storage. Be sure to update `%fileStorageClassName%` with your actual storage class.
+
+```yaml
+kubectl apply -f - <<EOF
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: llama-3-8b-pvc
+spec:
+  accessModes:
+    - ReadWriteMany
+  volumeMode: Filesystem
+  resources:
+    requests:
+      storage: 50Gi
+  storageClassName: %fileStorageClassName%
+EOF
+```
+
+### 2. Download the Model to the PVC
+  
+To download the model, export your Hugging Face token (`HF_TEST_TOKEN`) as an environment variable. Replace `%token%` with your actual token.
+
+
+```yaml
+export HF_TEST_TOKEN=%token%
+export MODEL=meta-llama/Meta-Llama-3-8B-Instruct
+
+curl -o download-model-to-pvc.yaml https://kserve.github.io/website/latest/modelserving/v1beta1/llm/huggingface/multi-node/download-model-to-pvc.yaml
+envsubst < download-model-to-pvc.yaml | kubectl create -f -
+```
+
+
+### 3. Create a ServingRuntime
+
+Apply the ServingRuntime configuration. Replace %TBD% with the path to your ServingRuntime YAML.
+
+```bash
+kubectl apply -f https://github.com/kserve/kserve/blob/master/config/runtimes/kserve-huggingfaceserver-multinode.yaml
+```
+
+### 4. Deploy the model 
+
+Finally, deploy the model using the following InferenceService configuration:
+
+```yaml
+kubectl apply -f - <<EOF
+apiVersion: serving.kserve.io/v1beta1
+kind: InferenceService
+metadata:
+  annotations:
+    serving.kserve.io/deploymentMode: RawDeployment
+    serving.kserve.io/autoscalerClass: external
+  name: huggingface-llama3
+spec:
+  predictor:
+    model:
+      runtime: kserve-huggingfaceserver-multinode
+      modelFormat:
+        name: huggingface
+      storageUri: pvc://llama-3-8b-pvc/hf/8b_instruction_tuned  
+    workerSpec: {}
+EOF
+```      
+
+## Check `InferenceService` status.
+
+To verify the status of your `InferenceService`, run the following command:
+
+```bash
+kubectl get inferenceservices huggingface-llama3
+```
+
+!!! success "Expected Output"
+    ```{ .bash .no-copy }
+    NAME                 URL                                                   READY   PREV   LATEST   PREVROLLEDOUTREVISION   LATESTREADYREVISION                          AGE
+    huggingface-llama3   http://huggingface-llama3.default.example.com                                          12m
```

Comment:
shouldn't `12m` be under AGE?

Reply:
It does not matter to me. And also, actually 405b model take 12 mins only for loading the model.